assets/js/base/components/quantity-selector/index.js

```diff
@@ -1,8 +1,14 @@
 /**
  * External dependencies
  */
+import { __, sprintf } from '@wordpress/i18n';
+import { speak } from '@wordpress/a11y';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import { useRef, useState, useEffect, useCallback } from '@wordpress/element';
+
+const KEYCODE_UP = 38;
+const KEYCODE_DOWN = 40;
```

Comment:
Could we use [`@wordpress/keycodes`](https://www.npmjs.com/package/@wordpress/keycodes) here?